From c6132a70ecfee585e7636f77b737eaec8170588d Mon Sep 17 00:00:00 2001
From: Randy Dunlap <rdunlap@infradead.org>
Date: Wed, 16 Mar 2022 13:41:46 -0700
Subject: [PATCH 01/18] sched/headers: ARM needs asm/paravirt_api_clock.h

Add <asm/paravirt_api_clock.h> for arch/arm/.

Fixes this build error:

In file included from ../kernel/sched/core.c:81:
../kernel/sched/sched.h:87:11: fatal error: asm/paravirt_api_clock.h: No such file or directory
   87 | # include <asm/paravirt_api_clock.h>

Fixes: 4ff8f2ca6ccd ("sched/headers: Reorganize, clean up and optimize kernel/sched/sched.h dependencies")
Signed-off-by: Randy Dunlap <rdunlap@infradead.org>
Cc: Ingo Molnar <mingo@kernel.org>
Cc: Russell King <linux@armlinux.org.uk>
Cc: linux-arm-kernel@lists.infradead.org
Cc: patches@armlinux.org.uk
Link: https://lore.kernel.org/r/20220316204146.14000-1-rdunlap@infradead.org
---
 arch/arm/include/asm/paravirt_api_clock.h | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 arch/arm/include/asm/paravirt_api_clock.h

diff --git a/arch/arm/include/asm/paravirt_api_clock.h b/arch/arm/include/asm/paravirt_api_clock.h
new file mode 100644
index 000000000000..65ac7cee0dad
--- /dev/null
+++ b/arch/arm/include/asm/paravirt_api_clock.h
@@ -0,0 +1 @@
+#include <asm/paravirt.h>

base-commit: 97add482e9033a9386baa8824fbea34d0cbfe663
-- 
2.35.1

